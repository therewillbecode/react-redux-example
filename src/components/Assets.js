import React from "react";

import AssetsListContainer from "../containers/AssetsListContainer";
import ReceiveAssetFormContainer from "../containers/ReceiveAssetFormContainer";

const Assets = props => (
  <div>
    <h2>Assets</h2>
    <ReceiveAssetFormContainer />
    <AssetsListContainer />
  </div>
);

export default Assets;
