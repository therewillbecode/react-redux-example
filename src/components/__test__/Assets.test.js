/* eslint-disable */
import React from "react";
import ReactShallowRenderer from "react-test-renderer/shallow";

import Assets from "../Assets";

describe("Assets", () => {
  it("renders correctly", () => {
    const renderer = new ReactShallowRenderer();
    renderer.render(<Assets />);
    const tree = renderer.getRenderOutput();
    expect(tree).toMatchSnapshot();
  });
});
