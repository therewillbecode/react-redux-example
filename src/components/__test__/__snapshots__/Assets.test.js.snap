// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Assets renders correctly 1`] = `
<div>
  <Connect(ReceiveAssetFormContainer) />
  <withRouter(Connect(AssetsListContainer)) />
</div>
`;
